visivo/commands/dbt_phase.py

```diff
@@ -0,0 +1,143 @@
+                    names.append(node["database"])
+                if "schema" in node:
+                    names.append(node["schema"])
+                names.append(node["name"])
+                model = {
+                    "name": f"dbt_{node['name']}",
+                    "sql": "select * from " + ".".join(names),
+                    "source": f"ref(dbt_{dbt_profile}_{dbt_target})",
+                }
+
+                models.append(model)
+
+    return models
+
+
+def dbt_phase(working_dir, output_dir, dbt_profile, dbt_target):
+    from visivo.logging.logger import Logger
+    from visivo.parsers.parser_factory import ParserFactory
+    from visivo.discovery.discover import Discover
+    from visivo.models.project import Dbt
+    import os
+    import click
+
+    discover = Discover(working_dir=working_dir, output_dir=output_dir)
+    parser = ParserFactory().build(
+        project_file=discover.project_file, files=discover.files
+    )
+    data = parser.merge_data_files()
+    if "dbt" in data and data["dbt"]:
+        dbt = Dbt(**data["dbt"])
+    else:
+        dbt = None
+
+    if dbt and dbt.enabled:
+        dbt_root = working_dir
+        if dbt.dbt_project_yml_location:
+            dbt_root = f"{working_dir}/{dbt.dbt_project_yml_location}"
+        dbt_project_file = f"{dbt_root}/dbt_project.yml"
+        if not os.path.exists(dbt_project_file):
+            raise click.ClickException(
+                f"dbt_project.yml file not found at '{dbt_project_file}'"
+            )
+
+        profiles_file = f"{working_dir}/profiles.yml"
+        if dbt.profiles_yml_location:
+            profiles_file = f"{working_dir}/{dbt.profiles_yml_location}/profiles.yml"
+
+        if not os.path.exists(profiles_file):
+            raise click.ClickException(
+                f"profiles.yml file not found at '{profiles_file}'"
+            )
+
+        Logger.instance().debug(f"Found dbt_project file: {dbt_project_file}")
+        Logger.instance().debug(f"Found profiles file: {profiles_file}")
+
+        import json
+        import ruamel.yaml
+
+        yaml = ruamel.yaml.YAML()
+        yaml.preserve_quotes = True
+
+        with open(dbt_project_file, "r") as file:
+            dbt_project = yaml.load(file)
+
+        with open(profiles_file, "r") as file:
+            profiles = yaml.load(file)
+
+        if not dbt_profile and "profile" in dbt_project:
+            dbt_profile = dbt_project["profile"]
+
+        if dbt_profile not in profiles:
+            raise click.ClickException(
+                f"Profile '{dbt_profile}' not found in profiles.yml"
+            )
+        if not dbt_target:
+            dbt_target = profiles[dbt_profile]["target"]
+
+        manifest_file = f"{dbt_root}/{dbt_project['target-path']}/manifest.json"
+        if not os.path.exists(manifest_file):
+            raise click.ClickException(
+                f"Manifest file not found at '{manifest_file}'. You might need to run dbt."
```

Comment:
It might be good to provide the command that they need to run.  
> f"Manifest file not found at '{manifest_file}'."+ "\n\tYou might need to run `dbt compile`."

I think in a follow up we should give people the option to include the `dbt compile` command in the dbt_phase every time automatically to avoid them getting out of sync. I think right now you could update your dbt project, not run any dbt commands and use an old manifest.json file